### PR TITLE
Task #16

### PR DESCRIPTION
**Pull request by Vaiki**

### CODE REVIEW

File: app/src/main/java/com/example/tmshw/RecyclerViewFragment.kt [hunk 1]
```diff
@@ -14,12 +14,13 @@ class RecyclerViewFragment : Fragment() {
     private var binding: FragmentRecyclerViewBinding? = null
```
Comment (OKurcheuskaya) [inline app/src/main/java/com/example/tmshw/RecyclerViewFragment.kt]: use laitenit

File: app/src/main/java/com/example/tmshw/RecyclerViewFragment.kt [hunk 2]
```diff
@@ -14,12 +14,13 @@ class RecyclerViewFragment : Fragment() {
...
     override fun onCreateView(
         inflater: LayoutInflater, container: ViewGroup?,
```
Comment (OKurcheuskaya) [inline app/src/main/java/com/example/tmshw/RecyclerViewFragment.kt:16]: https://developer.android.com/topic/libraries/view-binding

**Review by OKurcheuskaya — CHANGES_REQUESTED** [review]

File: app/src/main/java/com/example/tmshw/RecyclerViewFragment.kt (hunk 1 above)
Comment (Vaiki) [reply app/src/main/java/com/example/tmshw/RecyclerViewFragment.kt]: https://youtu.be/0nXXUzMyF8c?t=254

File: app/src/main/java/com/example/tmshw/RecyclerViewFragment.kt (hunk 2 above)
Comment (Vaiki) [reply app/src/main/java/com/example/tmshw/RecyclerViewFragment.kt:16]: я использую DataBinding, поэтому так https://developer.android.com/topic/libraries/data-binding/expressions

**Review by Vaiki — COMMENTED** [review]
fix

**Review by OKurcheuskaya — APPROVED** [review]

**Pull request merged**